pkg/cmd/pipelineresource/list_test.go

```diff
@@ -123,6 +123,18 @@ func TestPipelineResourceList(t *testing.T) {
 			args:      []string{"list", "-n", "test-ns-1", "-o", "jsonpath={range .items[*]}{.metadata.name}{\"\\n\"}{end}"},
 			wantError: false,
 		},
+		{
+			name:      "No Headers",
+			command:   command(t, pres, ns),
+			args:      []string{"list", "-n", "test-ns-1", "--no-headers"},
+			wantError: false,
+		},
```

Comment:
There should be two golden files, as you have added 2 tests, why 3 golden files?

Reply:
Oh I missed that. Let me update it.